CV Backend/src/main/java/com/qa/demo/security/SecurityConfiguration.java

```diff
@@ -0,0 +1,41 @@
+package com.qa.demo.security;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.factory.PasswordEncoderFactories;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import com.qa.demo.service.MongoUserDetailsService;
+
+@Configuration
+@EnableConfigurationProperties
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter
+{
+	@Autowired
+	MongoUserDetailsService userDetailsService;
+
+	protected void configure(HttpSecurity http) throws Exception
+	{
+		http.csrf().disable().authorizeRequests().antMatchers("/account/").permitAll().antMatchers("/account/**")
```

Comment:
format this monstrosity into something more readable pls

Reply:
done